airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -663,6 +765,16 @@ def execute(self, context) -> dict:
         self.log.info('Creating cluster: %s', self.cluster_name)
         hook = DataprocHook(gcp_conn_id=self.gcp_conn_id, impersonation_chain=self.impersonation_chain)
         # Save data required to display extra link no matter what the cluster status will be
```

Comment:
Could you move this comment to keep it next to the `sel.xcom_push()` which it refers to?